parlai/core/torch_ranker_agent.py

```diff
@@ -311,7 +311,7 @@ def _get_train_preds(self, scores, label_inds, cands, cand_vecs):
         else:
             _, ranks = scores.sort(1, descending=True)
         for b in range(batchsize):
-            rank = (ranks[b] == label_inds[b]).nonzero()
+            rank = (ranks[b] == label_inds[b].nonzero()[0]).nonzero()
```

Comment:
are these the changes for multi-class classification? does this still work with non-multiclass classification?